### PR TITLE
Use segmented control for mask selections

### PR DESCRIPTION
**Pull request by jy2wong**

### CODE REVIEW

**Comment by blanchardjeremy** [comment]
@jy2wong can you rebase your new commits that are unique to the PR on `main` now that the masks update is live there?

**Comment by beshaya** [comment]
Found a formatting oddity at around 1200px wide:
image

And again around 720px:
image

"Hover/Tap for descriptions" instruction probably needs to be changed depending on layout (hover doesn't work at mobile layout, tap doesn't work for desktop layout). Possibly this instruction can just be removed; the behavior feels quite intuitive and responsive to me.

I quite like how the description of the chosen mask is visible in mobile layout, maybe add something like that to desktop?

**Comment by jy2wong** [comment]
> Found a formatting oddity at around 1200px wide:
> image
> 
fixed (I think?)

> And again around 720px:
> image
> 
fixed

> "Hover/Tap for descriptions" instruction probably needs to be changed depending on layout (hover doesn't work at mobile layout, tap doesn't work for desktop layout). Possibly this instruction can just be removed; the behavior feels quite intuitive and responsive to me.
> 

> I quite like how the description of the chosen mask is visible in mobile layout, maybe add something like that to desktop?
done

**Comment by jy2wong** [comment]
This is ready for code review now.

File: src/components/calculator/controls/SegmentedControl.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  hideRisk?: boolean
+  className?: string
+  labelFactory?: (value: string) => string
+  variant?: string
+}> = (props) => {
+  const dataValue = props.id in props.data ? props.data[props.id] : ''
+  const activeValue = typeof dataValue === 'string' ? dataValue : ''
+  const { t } = useTranslation()
+  // 0.25 -> "1/4x"
+  function formatRiskMultiplierInternal(multiplier: number): string {
```
[the remark this answers is not in the document]
Comment (jy2wong) [reply src/components/calculator/controls/SegmentedControl.tsx:34]: This one is different from the one in SelectControl.tsx. The select one does 0.2 -> [1/5th the risk] whereas this one does 0.2 -> 1/5x.

File: src/components/calculator/ActivityRiskControls.tsx [hunk 2]
```diff
@@ -61,26 +63,77 @@ export const ActivityRiskControls: React.FunctionComponent<{
           </Trans>
         </div>
       )}
-      <SelectControl
+      <SegmentedControl
```
[the remark this answers is not in the document]
Comment (jy2wong) [reply src/components/calculator/ActivityRiskControls.tsx:123]: done

[the remark this answers is not in the document]
Comment (jy2wong) [reply src/components/calculator/ActivityRiskControls.tsx]: done

**Review by beshaya — APPROVED** [review]
lgtm

**Pull request merged**